### ISSUE TITLE
Add leather furniture.

### ISSUE THREAD

**Issue by Aimless-dude** [CleverRaven/Cataclysm-DDA#78433]
### Is your feature request related to a problem? Please describe.

We have fabric armchairs in game, so why not also have leather ones? Also would be a great source of leather.

### Solution you would like.

Have a chance for an armchair to be made from leather instead of cotton- presumably with some coding magic to make sure couches are all one or the other. I imagine they'd be identical except for the leather one dropping leather instead of cotton.

**Comment by PGR-14** [comment]
Makes sense to me, but one small problem: That _isn't_ leather. 9/10 times, it's just faux leather. IDK how that's precedented in other places, but it _should_ be fine.
